proof-systems/src/darlin/accumulators/dlog.rs

```diff
@@ -232,14 +232,6 @@ impl<G: AffineCurve, D: Digest> ItemAccumulator for DLogItemAccumulator<G, D> {
         // Sample a new challenge z
         let z = fs_rng.squeeze_128_bits_challenge::<G::ScalarField>();
 
-        // Collect GFinals from the accumulators
```

Comment:
Even though not related to these lines: 
let us setup the fs_rng as Marlin does (using a PROTOCOL_NAME).

Reply:
call the protocol "ANNAGIULIA" :)